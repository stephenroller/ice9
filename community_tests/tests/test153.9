#  fa & proc overriding prior loop variable defns					succeed

var i : int

proc foo(i : int)

	fa i := 0 to 1 ->
		writes i;
		writes "";
	af

    write "";
end

fa i := 0 to 1 ->
	write i;
	foo(i);
af
