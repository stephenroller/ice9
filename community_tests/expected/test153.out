Loading file: generated/test153.tm
0 
0 1 
1 
0 1 
Number of instructions executed = 270
